### PR TITLE
Add Property Expression FAT test to mpConfig-2.0

### PR DESCRIPTION
**Pull request by Joseph-Cass**
#build

### CODE REVIEW

File: dev/io.openliberty.microprofile.config.2.0.internal_fat/publish/servers/Config20Server/bootstrap.properties [hunk 1]
```diff
@@ -4,3 +4,5 @@ io.openliberty.microprofile.config.internal_fat.apps.brokenInjection.ConfigUnnam
 io.openliberty.microprofile.config.internal_fat.apps.brokenInjection.ConfigUnnamedMethodInjectionBean.SIMPLE_KEY6=VALUE6
 noConverterKey=value
 org.jboss.logging.provider=slf4j
+value1DefinedInBootstrapProperties=value1
+value2DefinedInBootstrapProperties=value2
```
Comment (Emily-Jiang) [inline dev/io.openliberty.microprofile.config.2.0.internal_fat/publish/servers/Config20Server/bootstrap.properties:8]: Can you define the one in `microprofile-config.properties` instead?

Comment (Joseph-Cass) [reply dev/io.openliberty.microprofile.config.2.0.internal_fat/publish/servers/Config20Server/bootstrap.properties:8]: Good idea. I'll give that a try :)

**Comment by Joseph-Cass** [comment]
Resolves #14525

File: dev/io.openliberty.microprofile.config.2.0.internal_fat/publish/servers/Config20Server/bootstrap.properties (hunk 1 above)
Comment (Joseph-Cass) [reply dev/io.openliberty.microprofile.config.2.0.internal_fat/publish/servers/Config20Server/bootstrap.properties]: How does this look: https://github.com/OpenLiberty/open-liberty/pull/14823/commits/c3296348f6cd8ce48a0f24b7db82c38f4b4b7651 ?

**Comment by Joseph-Cass** [comment]
#build

**Comment by Joseph-Cass** [comment]
Build had an RTC blip. Trying again #build

File: dev/io.openliberty.microprofile.config.2.0.internal_fat/publish/servers/Config20Server/bootstrap.properties [hunk 2]
```diff
@@ -4,3 +4,4 @@ io.openliberty.microprofile.config.internal_fat.apps.brokenInjection.ConfigUnnam
 io.openliberty.microprofile.config.internal_fat.apps.brokenInjection.ConfigUnnamedMethodInjectionBean.SIMPLE_KEY6=VALUE6
 noConverterKey=value
 org.jboss.logging.provider=slf4j
+value1DefinedInBootstrapProperties=value1
```
Comment (Emily-Jiang) [inline dev/io.openliberty.microprofile.config.2.0.internal_fat/publish/servers/Config20Server/bootstrap.properties:7]: Can you update this name to something like the following and also define this in microprofile-config.properties to test that the override capabilities? Sorry, I forgot to mention last week.
`value1DefinedInTwoPlaces=value1`

**Review by Emily-Jiang — APPROVED** [review]
LGTM! Thanks @Joseph-Cass !

**Comment by Joseph-Cass** [comment]
#run-libby-bot

**Pull request merged**